OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/optimobile/Topic.java

```diff
@@ -0,0 +1,33 @@
+package com.optimove.android.optimobile;
+
+import java.util.List;
+
+public class Topic {
+    private final String id;
+    private final String name;
+    private final String description;
+    private final List<Integer> subscribedChannels;
+
+    Topic(String id, String name, String description, List<Integer> subscribedChannels) {
+        this.id = id;
+        this.name = name;
+        this.description = description;
+        this.subscribedChannels = subscribedChannels;
```

Comment:
this class is public and will be used by customers. How would they know that 489 is push? i'd just have some List<Enum> and not drag numbers along. somewhere close to GET / SET we would map